From b9fcfd6e79f3e5169e1ad82d24c6ca2f56ab4e48 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Sun, 10 Oct 2021 22:56:59 +0200
Subject: [PATCH 2/2] server: Fix missing include for uid_t

Explicitly include <sys/types.h> to make uid_t available to users of
server/security.h.

This is a problem on musl, and possibly other non-glibc systems.

Fixes: c954e5b9e6a6 ("server: Avoid using wine/port.h.")
Upstream-Status: Submitted [https://bugs.winehq.org/show_bug.cgi?id=51860]
Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 server/security.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/server/security.h b/server/security.h
index 5388bc9c4195..39e7218ec5fa 100644
--- a/server/security.h
+++ b/server/security.h
@@ -21,6 +21,8 @@
 #ifndef __WINE_SERVER_SECURITY_H
 #define __WINE_SERVER_SECURITY_H
 
+#include <sys/types.h> /* for uid_t */
+
 extern const LUID SeIncreaseQuotaPrivilege;
 extern const LUID SeSecurityPrivilege;
 extern const LUID SeTakeOwnershipPrivilege;
-- 
2.33.0

